XenAdmin/Dialogs/OptionsPages/ConnectionOptionsPage.cs

```diff
@@ -92,6 +87,19 @@ private void build()
             if (Registry.ProxyAuthenticationEnabled)
             {
                 AuthenticationCheckBox.Checked = Properties.Settings.Default.ProvideProxyAuthentication;
+                
+                switch ((HTTPHelper.ProxyAuthenticationMethod)Properties.Settings.Default.ProxyAuthenticationMethod)
+                {
+                    case HTTPHelper.ProxyAuthenticationMethod.Basic:
+                        BasicRadioButton.Checked = true;
+                        break;
+                    case HTTPHelper.ProxyAuthenticationMethod.Digest:

```

Comment:
We decided to leave this as is to make any future extension/change easier (w.r.t. authentication schemes).